src/platforms/linux/daemon/wghelper.cpp

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include "wghelper.h"
+#include "wgquickprocess.h"  // source of WG_INTERFACE
+
+#include <QString>
+#include <QByteArray>
+#include <QStringList>
+
+#include "logger.h"
+#include "loghandler.h"
+
+// Import wireguard C library
+#if defined(__cplusplus)
+extern "C" {
+#endif
+#include "../../3rdparty/wireguard-tools/contrib/embeddable-wg-library/wireguard.h"
+#if defined(__cplusplus)
+}
+#endif
+
+namespace {
+Logger logger(LOG_LINUX, "WireguardHelper");
+}
+
+bool WireguardHelper::interface_exists() {
+  // Also confirms it is wireguard
+  QStringList wg_devices = WireguardHelper::current_wireguard_devices();
+  return wg_devices.contains(WG_INTERFACE);
+};
+
+QStringList WireguardHelper::current_wireguard_devices() {
```

Comment:
1. Add a `// static` to know it's a static method.
2. currentWireguardDevices()